caikit/interfaces/runtime/data_model/package.py

```diff
@@ -0,0 +1,21 @@
+# Copyright The Caikit Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+Package constant for all runtime service data model objects
+"""
+
+# Local
+from caikit.core.data_model import CAIKIT_DATA_MODEL
+
+RUNTIME_PACKAGE = f"{CAIKIT_DATA_MODEL}.runtime"
```

Comment:
We were missing a common constant for the `interfaces.runtime` package. As such, there's [inconsistency in the packages](https://github.com/caikit/caikit/issues/700) which I opted not to fix at the moment to avoid an API breaking change.